addons/sale/static/src/js/product_catalog/sale_order_line/sale_order_line.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
+    <t t-name="sale.ProductCatalogSOL" owl="1">
+        <div class="my-2 pt-3 border-top d-flex justify-content-end">
+            <t t-portal="'#product-'+props.productId">
```

Comment:
might be useful to add a comment here to explain the use of the t-portal (you explained it to me, but it should be clear by itself).